app/main.py

```diff
@@ -1 +1,53 @@
-#  write your code here
+from typing import Any
+
+
+class Animal:
+    def __init__(
+        self,
+        name: str,
+        appetite: int,
+        is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            return self.appetite
```

Comment:
The `feed` method should update the `is_hungry` attribute to `False` after feeding the animal. This ensures that the animal's state is correctly updated after being fed.